contrib/plugins/qfs_monitor.cc

```diff
@@ -131,6 +131,7 @@ extern "C" void reportStatus(
 {
     int pid = getpid();
     string logFilePath = getLogPath();
+    logFilePath += "/";
```

Comment:
nit: If you're always going to do this, then I would remove it from the default value.